AirWallex/Airwallex/Sources/AWXPaymentMethodListViewController.m

```diff
@@ -245,7 +245,7 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
     
     if (indexPath.section == 1) {
         AWXPaymentMethodType *paymentMethodType = self.availablePaymentMethodTypes[indexPath.row];
-        Class class = ClassToHandleFlowForPaymentMethodType(paymentMethodType.name);
+        Class class = ClassToHandleFlowForPaymentMethodType(paymentMethodType);
         if (class == nil) {
```

Comment:
our FE are using sentry to monitoring error on client side.
I'm considering whether we can also use sentry to monitor such cases so that we can be notified when this failure happens?